sync/class.jetpack-sync-module-themes.php

```diff
@@ -65,8 +69,9 @@ public function sync_network_allowed_themes_change( $option, $value, $old_value,
 			 *
 			 * @param mixed $newly_disabled_themes, Array of info about network disabled themes
 			 * @param mixed $all_enabled_theme_slugs, Array of slugs of all enabled themes
+			 * @param bool $is_theme_deletion, Whether a theme was deleted Since 5.3
 			 */
-			do_action( 'jetpack_network_disabled_themes', $newly_disabled_themes, $all_enabled_theme_slugs );
+			do_action( 'jetpack_network_disabled_themes', $newly_disabled_themes, $all_enabled_theme_slugs, $is_theme_deletion );
```

Comment:
A theme deleted sync action is already being sent independently. This prevents the two activities from appearing in the activity log.